Makefile

```diff
@@ -1,6 +1,9 @@
 .PHONY: release
 
-version=$(shell grep -o '"version"\s*:\s*"\S*"' manifest.json | sed -e 's/.*"\([0-9]\S*\)".*/\1/')
+all: send_later.xpi
+clean: ; -rm -f send_later.xpi
+
+version=$(shell grep -o '"version"\s*:\s*"\S*"' manifest.json | sed -e 's/.*"\([0-9].*\)".*/\1/')
```

Comment:
Just to be pedantic, maybe replace `\S` with the posix equivalent `[^[:space:]]`

Reply:
I don't think it's necessary to make the regexp more complicated like that because we're the maintainers of manifest.json so we know exactly what its format will be and we therefore know the simpler regexp will dtrt.

Also, named character classes aren't supported by every version of sed either.